code/controllers/subsystems/ambience.dm

```diff
@@ -50,15 +48,21 @@ SUBSYSTEM_DEF(ambience)
 
 	if(lit)
 
-		// If we're using area lighting, we're indoors, so shouldn't use level data ambience.
-		if(override_light_power)
-			set_ambient_light(override_light_color || COLOR_WHITE, override_light_power)
-			return TRUE
-
 		// Grab what we need to set ambient light from our level handler.
 		var/datum/level_data/level_data = SSmapping.levels_by_z[z]
+
+		// Check for daycycle ambience.
+		if(level_data.daycycle_id)
+			var/datum/daycycle/daycycle = SSdaycycle.get_daycycle(level_data.daycycle_id)
+			var/new_power = daycycle?.current_period?.power
+			if(!isnull(new_power))
+				if(new_power > 0)
+					set_ambient_light(daycycle.current_period.color, clamp(new_power + ambient_light_modifier, 0, 1))
```

Comment:
Actually, I think this should probably `clear_ambient_light()` if it's equal to zero?